libvirt/tests/src/virtual_network/migrate/migrate_with_ethernet_interface.py

```
@@ -0,0 +1,189 @@
+    server_ip = params["server_ip"] = params.get("migrate_dest_host")
+    server_user = params.get("server_user")
+    server_pwd = params.get("server_pwd") or params.get("migrate_dest_pwd")
+    outside_ip = params.get("outside_ip")
+    bridge_name = params.get("bridge_name", "br0")
+    tap_name = params.get("tap_name", "mytap0")
+    tap_mtu = params.get("tap_mtu")
+    cancel_migration = params.get_boolean("cancel_migration")
+    migrate_vm_back = params.get_boolean("migrate_vm_back", True)
+
+    dest_uri = params.get("virsh_migrate_desturi")
+    iface_attrs = eval(params.get("iface_attrs", "{}"))
```

Ruff (0.14.1)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)